api.php

```diff
@@ -1,151 +1,82 @@
 <?php
```

Comment:
Github fails to detect that there is basically no change in the PHP API. The old file `api.php` is not rewritten, but moved to `api_PHP.php`. This `api.php` is an entirely new file.